calicoctl/calico_ctl/node.py

```diff
@@ -612,8 +612,16 @@ def node_remove(remove_endpoints):
                         "in the normal way.")
         sys.exit(1)
 
+    # Remove the veths, and release all IPs associated with the endpoint.
+    ips = set()
     for endpoint in endpoints:
         remove_veth(endpoint.name)
+        ips |= {net.ip for net in endpoint.ipv4_nets}
```

Comment:
This isn't a common operator. Can you add a comment saying what this does?

I assume this is setting the `ips` set to include the set of `ips` in both `endpoint.ipv4_nets` and `endpoint.ipv6_nets`, combining all unique IPs into one set to remove.